### PR TITLE
Auto assigning project to github Issues and Feature request

### PR DESCRIPTION
**Pull request by evertonlperes**
### Changes
1. Included Bug Report and Feature request on `config.yml`
2. Reused existing templates

Goal: This change will auto assign `Stripey` project in every issue/feature opened on RD Issues tab.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 blank_issues_enabled: true
 contact_links:
+- name: Report a bug
```
Comment (evertonlperes) [inline .github/ISSUE_TEMPLATE/config.yml:3]: @jandubois 
Renamed the `name` tag from `config.yml` in order to be different from `bug_report.yml`, but I have a feeling GitHub will display both entries on Issues tab:
* A button from `./ISSUE_TEMPLATE/bug_report.yml`
* and a button from the `config.yml`

**Comment by jandubois** [comment]
I don't understand why you are adding those links; what are you trying to accomplish? They are redundant with the issue templates, and will not work for users that don't have write access to this repo because they are not allowed to assign labels. It has to be done through an issue template.

**Review by jandubois — CHANGES_REQUESTED** [review]
Use issue templates, not contact links, to file bugs and issues.

**Comment by evertonlperes** [comment]
Closing this issue.
I'll start a simple automation on GHA for it.

**Pull request closed without merging**